website/content/en/preview/concepts/deprovisioning.md

```diff
@@ -128,6 +129,29 @@ data:
   ...
 ```
 
+## Drift
```

Comment:
I think we need some language around how drift is handled with respect to other terminations (e.g. consolidation, etc)